src/sh/BUILD

```diff
@@ -0,0 +1 @@
+shell_sources()
```

Comment:
notable: I just added `src/sh` for the first time

the rust dockerfile context starts in `src/` so it seemed a reasonable place to put it